inference-engine/src/mkldnn_plugin/config.cpp

```diff
@@ -21,16 +21,26 @@ namespace MKLDNNPlugin {
 using namespace InferenceEngine;
 
 Config::Config() {
-#if (defined(__APPLE__) || defined(_WIN32))
-#if (IE_THREAD == IE_THREAD_TBB || IE_THREAD == IE_THREAD_TBB_AUTO) && (TBB_INTERFACE_VERSION >= 11100)
-    // If we sure that TBB has NUMA aware API part.
-    streamExecutorConfig._threadBindingType = InferenceEngine::IStreamsExecutor::NUMA;
-#else
-    streamExecutorConfig._threadBindingType = InferenceEngine::IStreamsExecutor::NONE;
-#endif
-#else
+    // this is default mode
     streamExecutorConfig._threadBindingType = InferenceEngine::IStreamsExecutor::CORES;
-#endif
+
+    // for the TBB code-path, additional configuration depending on the OS and CPU types
+    #if (IE_THREAD == IE_THREAD_TBB || IE_THREAD == IE_THREAD_TBB_AUTO)
+        #if TBB_INTERFACE_VERSION >= 11100 // NUMA-aware TBB
+            #if defined(__APPLE__) || defined(_WIN32)
+            // 'CORES' is not implemented for Win/MacOS, so the 'NUMA' is default
+            streamExecutorConfig._threadBindingType = InferenceEngine::IStreamsExecutor::NUMA;
```

Comment:
May be if there is no NUMA nodes InferenceEngine::IStreamsExecutor::NONE should be used